app/utils/ipfs.js

```diff
@@ -3,20 +3,41 @@ import bs58 from 'bs58';
 import { create } from 'ipfs-http-client';
 
 import { ApplicationError } from './errors';
+import {
+  callService,
+  SAVE_FILE_SERVICE,
+} from 'utils/web_integration/src/util/aws-connector';
+import { Web3Storage } from 'web3.storage/dist/bundle.esm.min.js';
 
-export function getIpfsApi() {
+export function ipfsApi() {
   return create(process.env.IPFS_API_URL);
 }
 
-function getIpfsApiTheGraph() {
+function ipfsApiTheGraph() {
   return create(process.env.IPFS_API_URL_THE_GRAPH);
 }
 
+function web3StorageApi() {
+  const token = process.env.WEB3_STORAGE_API_TOKEN;
+
+  if (!token) {
+    throw new Error(
+      'A token is needed. You can create one on https://web3.storage',
```

Comment:
Change message to `WEB3_STORAGE_API_TOKEN environment variable is not defined`

Reply:
done